docs/semgrep-app/scm.md

```diff
@@ -10,14 +10,24 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrating Semgrep into source code management (SCM) tools
 
-Semgrep App's Team tier supports repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. Create multiple integrations in different self-hosted or custom servers.
+Semgrep App's Team and Enterprise tiers support repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. When referring to SCM tools, this term means explicitly GHE or GLSM plans in this document.

```

Comment:
(Feel free to ignore and resolve after reading.) Still not sure about the word "both" here, but feel free to ignore me, please. As mentioned previously, there are many SCM tools, so saying: We support both - sounds like a logical error. Because there are not just two. @s-santillan

Reply:
[https://www.merriam-webster.com/dictionary/both](https://www.merriam-webster.com/dictionary/both)



Definition of both (Entry 2 of 3)
—used as a function word to indicate and stress the inclusion of each of two or more things specified by coordinated words, phrases, or clauses

The use of "both" is about **stressing inclusion** not excluding other .tools.

Reply:
Interesting linguistic issue here. Just a heads up here - @s-santillan yes to Merrian-Webster, but no to word both in this context. Bot legs, arms - because in a context of a human body, we are considering only two legs or arms. But here you have many SCMs on the market, not just two. The terms seems to be quite broad, it can mean both Git or Mercurial and others, but also services which use them, such as GitHub or GitLab.

Imagine: There are tools called A, B, C, D, E. Documentation would say: Oh hey, we support both A and B!
But such statement is a logical fallacy. There are not two (both) A and B, there are many more such tools, in this example also C, D, and E.

So I would really remove both as it carries strange meaning.